scripts/start/prepare_and_run.R

```diff
@@ -998,28 +1001,30 @@ run <- function(start_subsequent_runs = TRUE) {
         message("  Number of iterations: ",
                          as.numeric(readGDX(gdx="fulldata.gdx", "o_iterationNumber", format = "simplest")))
         message("  Modelstat: ", as.numeric(readGDX(gdx="fulldata.gdx", "o_modelstat", format="simplest")),
-                " (see https://www.gams.com/mccarlGuide/modelstat_tmodstat.htm).")
+                " (see https://github.com/remindmodel/remind/blob/develop/tutorials/10_DebuggingREMIND.md).")
       }
       logStatus <- grep("*** Status", readLines("full.log"), fixed = TRUE, value = TRUE)
       message("  full.log states: ", paste(logStatus, collapse = ", "))
       if (! all("*** Status: Normal completion" == logStatus)) stoprun <- TRUE
     }
   }
 
-  if ( file.exists("full.lst")) {
-    message("Infeasibilities extracted from full.lst with nashstat -F:")
-    command <- paste("li=$(nashstat -F | wc -l); cat",
-               "<(if (($li < 2)); then echo no infeasibilities found; fi)",
-               "<(if (($li > 1)); then nashstat -F | head -n 2; fi)",
-               "<(if (($li > 4)); then echo ... $(($li - 3)) infeasibilities omitted, show all with nashstat -a ...; fi)",
-               "<(if (($li > 2)); then nashstat -F | tail -n 1; fi)")
+  if (identical(cfg$gms$optimization, "nash") && file.exists("full.lst")) {
+    message("\nInfeasibilities extracted from full.lst with nashstat -F:")
+    command <- paste(
+      "li=$(nashstat -F | wc -l); cat",
+      "<(if (($li < 2)); then echo no infeasibilities found; fi)",
+      "<(if (($li > 1)); then nashstat -F | head -n 2 | sed -r 's/\\x1B\\[([0-9]{1,2}(;[0-9]{1,2})?)?[m|K]//g'; fi)",
+      "<(if (($li > 4)); then echo ... $(($li - 3)) infeasibilities omitted, show all with nashstat -a ...; fi)",
+      "<(if (($li > 2)); then nashstat -F | tail -n 1 | sed -r 's/\\x1B\\[([0-9]{1,2}(;[0-9]{1,2})?)?[m|K]//g'; fi)")
```

Comment:
Would it be possible to make the dependency on `nashstat` softer? At the moment, there is an error if `nashstat` is not found, but in principle, REMIND can be run somewhere else than on the cluster, and `nashstat` is proprietary secret-sauce which is not available for mere mortals outside PIK.

Reply:
yes, that is what I did.